fhir-persistence-cassandra/src/main/java/com/ibm/fhir/persistence/cassandra/CosPropertyGroupAdapter.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * (C) Copyright IBM Corp. 2021
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.ibm.fhir.persistence.cassandra;
+
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import com.ibm.fhir.config.PropertyGroup;
+
+/**
+ * Provides a facade on top of the fhir-server-config PropertyGroup structure
+ * to simplify access to configuration elements we need for connecting to
+ * COS
+ */
+public class CosPropertyGroupAdapter {
```

Comment:
is this a temporary stash, or does this really belong in the cassandra module (and the cassandra package)?
if it truly belongs in the same module as the cassandra impl, maybe a more generic name like "fhir-persistence-offload" is in order?
i know the main focus of this PR is the refactor, so probably fair to revisit this later

Reply:
ok, i see a class by the same name under the `fhir-persistence-cos` project, so I'm assuming this one is stale and should be deleted.